website/docs/getting-started/setup/instance-configuration/custom-domain/README.md

```diff
@@ -83,8 +83,13 @@ To host the Appsmith DigitalOcean droplet on a custom domain, you'll need to sel
 
 ![](/img/custom\_domain.jpeg)
 
-It redirects you to a new page. Add your domain name there. Once that's done, it'll give you records of the name servers. Copy the details of the NS (name servers). Use the custom name server's configuration on your domain provider. Sometimes, it might take up to 24-48 hours for this to go live!
+It redirects you to a new page. Add your domain name there. Once that's done, it'll give you records of the name servers. Copy the details of the NS (name servers). Use the custom name server's configuration on your domain provider. Sometimes, it might take up to 24 or 48 hours for this to go live.
 
 :::tip
 Your Appsmith instance should be available at [https://appsmith.mydomain.com](https://appsmith.mydomain.com) with automatic certificate provisioning and renewals.
 :::
+
+## Kubernetes
+
+The `APPSMITH_CUSTOM_DOMAIN` environment variable is no longer used for configuring TLS for a Kubernetes installation of Appsmith. To setup TLS on your Kubernetes installation of Appsmith, refer to the instructions [here](Configure the `APPSMITH_CUSTOM_DOMAIN field` in your [instance configuration](../) with your custom domain).
```

Comment:
@sum35h - Where should the cross-reference to?

```
refer to the instructions [here](Configure the `APPSMITH_CUSTOM_DOMAIN field` in your [instance configuration](../) with your custom domain).
```

Should this section be added to Docker/K8s or should the `Docker/K8s` be renamed to Docker?

Reply:
That instruction should point to getting-started/setup/installation-guides/kubernetes#configure-tls

Reply:
Should this section be added to Docker/K8s or should the Docker/K8s be renamed to Docker? Sorry I did not understand this properly.

Reply:
We already have a section on this page [Docker/K8s](https://docs.appsmith.com/getting-started/setup/instance-configuration/custom-domain#dockerk8s). Do we need to add the details for Kubernetes added in this PR to the above section or should we rename it to Docker and have a separate section for Kubernetes.